core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1888,6 +1888,12 @@ public SqlSyntax getSyntax() {
   public static final SqlFunction CURRENT_TIMESTAMP =
       new SqlAbstractTimeFunction("CURRENT_TIMESTAMP", SqlTypeName.TIMESTAMP);
 
+  /**
+   * The <code>CURRENT_DATETIME [(<i>precision</i>)]</code> function.
+   */
+  public static final SqlFunction CURRENT_DATETIME =
+      new SqlAbstractTimeFunction("CURRENT_DATETIME", SqlTypeName.TIMESTAMP);
+
```

Comment:
Does `CURRENT_DATETIME` belong to the SQL standard ?

Reply:
No, I realized there is a better way to do this. I'll remove this change.